html_text/html_text.py

```diff
@@ -7,6 +7,9 @@
 import parsel
 
 
+NEWLINE_TAGS = ['li', 'dd', 'dt', 'dl', 'ul', 'ol']
+DOUBLE_NEWLINE_TAGS = ['title', 'p', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6']
```

Comment:
These lists are used for lookups; even though they're short, I think it is cleaner and faster to have them as sets. 

```
In [1]: x = ['title', 'p', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6']
In [2]: x_set = {'title', 'p', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6'}
In [3]: %timeit 'foo' in x
162 ns ± 0.398 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

In [4]: %timeit 'foo' in x_set
39.8 ns ± 0.153 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```

Reply:
It'd also be nice to allow overriding these double_newline_tags and newline_tags in extract_text; these constants would then just be defaults (use frozenset instead of set if you do so).